apps/web/pages/booking/[uid].tsx

```diff
@@ -165,10 +165,13 @@ export default function Success(props: SuccessProps) {
     );
   }
 
+  const evtName = !eventType.isDynamic
+    ? (props.dynamicEventName as string) || props.eventType.eventName
```

Comment:
Is there any situation that the `props.dynamicEventName` is not empty when `eventType.isDynamic` is true?
If not, I'd suggest to remove it.
```suggestion
? props.eventType.eventName
```

Reply:
Ohk 👍 
If the event is not dynamic event, then dynamicEvent's value is same as eventType.eventName's value, so I have removed as you suggested.